lib/core/mock/language.service.mock.ts

```diff
@@ -0,0 +1,46 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+import { LanguageItem } from '../services/language-item.interface';
+
+@Injectable()
+export class LanguageServiceMock {
```

Comment:
could you please introduce a new interface LanguageServiceInterface and implement it? In this way we are sure that the mock is in sync with the real service. 

Of course also LanguageService must implement the new interface LanguageServiceInterface
```suggestion
export class LanguageServiceMock implements LanguageServiceInterface {
```

Reply:
Added the Interface in new push, both LanguageService and LanguageServiceMock implement it.